### ISSUE TITLE
Back buttons should be identified as navigation elements for accessibility

### ISSUE THREAD

**Issue by hspinks** [WorldHealthOrganization/app#924]
Accessibility Audit via @avkvirtru 

**Problem:** back buttons are inaccessible because they are not identified as navigation elements. This problem exists for all pages that have a back button.

**Acceptance Criteria:** Header should be a separate element with the title of the screen. Back button should be a separate element.

---
- [x] Searched the [existing issues](https://github.com/WorldHealthOrganization/app/issues) to ensure you are not creating a duplicate.
- [x] Followed the [Contributor Guidelines](https://github.com/WorldHealthOrganization/app/blob/master/docs/CONTRIBUTING.md).
